packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -197,6 +206,30 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
         // Intentionally empty.
     }
 
+    // TODO: I don't like having so many return args but I also don't like
+    // wrapping them into a struct
+    function resourceConfig() public view override returns (SystemConfig.ResourceConfig memory) {
```

Comment:
I'm a bit unclear why this function needs to go into the Portal when it is only used in the ResourceMetering code?